docs/configuration/upstream/propose_downstream.md

```diff
@@ -32,7 +39,63 @@ jobs:
 - job: propose_downstream
   trigger: release
   dist_git_branches:
-    - f35
+    - f39
 ```
 
-This config would update Fedora Rawhide and Fedora 35 dist-git branches.
+This config would update Fedora Rawhide and Fedora 39 dist-git branches.
+
+
+
+## Syncing the release to CentOS Stream
+
+Recently, we have implemented support for `propose-downstrem` also for CentOS Stream. This means that you can now configure
+the upstream release to trigger release syncing the same way as for Fedora to CentOS Stream dist-git repositories
+hosted in https://gitlab.com/redhat/centos-stream/rpms.
```

Comment:
Do we want to provide a disclaimer/warning that it might not work as expected?

Reply:
Also, what about asking for feedback and reporting issues so we can improve the CentOS integration?

Reply:
good point, I will add a hint or something like that